FirebaseSessions/Sources/Public/SessionsSubscriber.swift

```diff
@@ -28,7 +28,7 @@ public protocol SessionsSubscriber {
 /// whenever the Session changes
 @objc(FIRSessionDetails)
 public class SessionDetails: NSObject {
-  var sessionId: String?
+  @objc public var sessionId: String?
```

Comment:
FYI I moved this to a separate PR that I hope we could merge early because this fix is needed for my PR to work as well

Reply:
Awesome - thanks for doing that